config/initializers/okcomputer.rb

```diff
@@ -1,3 +1,4 @@
 OkComputer.mount_at = "healthcheck"
 
 OkComputer::Registry.register "database", OkComputer::ActiveRecordCheck.new
+OkComputer::Registry.register "Commit SHA", OkComputer::AppVersionCheck.new
```

Comment:
```suggestion
OkComputer::Registry.register "sha", OkComputer::AppVersionCheck.new
```

So we can access it at `/healthchecks/sha`